src/storagesync/azext_storagesync/_help.py

```diff
@@ -0,0 +1,289 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['storagesync'] = """
+    type: group
+    short-summary: Manage Azure File Sync.
+"""
+
+helps['storagesync storage-sync-service'] = """
+    type: group
+    short-summary: Manage storage sync service.
+"""
+
+helps['storagesync storage-sync-service create'] = """
+    type: command
+    short-summary: Create a new storage sync service.
+    examples:
+      - name: Create a new storage sync service "SampleStorageSyncService" in resource group 'SampleResourceGroup'.
+        text: |-
+               az storagesync storage-sync-service create --resource-group "SampleResourceGroup" \\
```

Comment:
better to use `MyResourceGroup`. Align with other examples.

Reply:
It's generated by codegen, and it should be from swagger samples, in the future, we should just use the rg from swagger without manual update.

---
In reply to: [392867347](https://github.com/Azure/azure-cli-extensions/pull/1389#discussion_r392867347) [](ancestors = 392867347)